lib/perilune/configuration.rb

```diff
@@ -2,13 +2,22 @@
 
 module Perilune
   class Configuration
-    def queue_name
-      :default
+    attr_accessor :queue_name, :stat_driver
+
+    def initialize
+      @queue_name = :default
+      @stat_driver = ::Trifle::Stats::Driver::Redis.new
+      @stat_track_ranges = %i[hour day week month quarter year]
+      @stat_time_zone = 'GMT'
+      @stat_beginning_of_week = :monday
```

Comment:
I would leave in initialize only those that we plan to override and then leave rest of the configuration in a `stats_driver_config`.
```suggestion
    def initialize
      @queue_name = :default
      @stat_driver = ::Trifle::Stats::Driver::Redis.new
```